src/containers/NoteForm/NoteForm.js

```diff
@@ -77,6 +76,10 @@ export class NoteForm extends Component {
     const { items } = this.state;
     const { isEdit, editNote, postNote } = this.props;
     const url = isEdit ? `http://localhost:3001/api/v1/notes/${id}` : 'http://localhost:3001/api/v1/notes'
+    
+    if (items[items.length - 1].description === '') {
+      items.pop()
+    }
```

Comment:
I think we might be manipulating state directly here

Reply:
Fixed now sir